routes/items/item_response.js

```diff
@@ -17,6 +18,37 @@ const filteredItemsQuery = (Item, query, user_id) => ({ items, tree }) => {
   }
 }
 
+const selectedItemsQuery = item_id => (
+  { order: [['createdAt', 'ASC']], where: {
+    $or: [
+      { id: item_id },
+      { parent_id: item_id }
+    ]
+  }, FETCH_ATTRIBUTES }
+)
+
+const filterSelectedItems = (Item, query, user_id) => ({ items, tree }) => {
+  const where = Object.assign( {}, whereSearch( query ), whereCompleted( query), { user_id } )
+
+  if( Object.keys( where ).length === 1 ) {
+    return { items, tree }
+  } else {
+    return Item.findAll({ where, attributes: [ 'id' ] })
+      .then( pruneTree( items, tree ))
+  }
+}
+
+const findBreadcrumbNav = ({ items, tree }) => {
+  const ids = tree.findPathTo( item_id )
```

Comment:
`item_id` is not defined in this context - it probably needs to be passed in:
```js
const findBreadcrumbNavPleaseRename => item_id => ({ items, tree }) => {
```
And then it could be called like this in the promise chain:
```js
  .then( findBreadcrumbNavPleaseRename( item_id ) ) // the item_id would be tree.root.id for the first route
```